pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -123,6 +123,25 @@ svg {
     z-index: 2;
 }
 
+// Remove the default :focus-visible outline from elements that
+// receive the :focus-visible class also when focused via the mouse
+// and already have a visible :focus style.
+//
+// The default :focus-visible rendering differs from browser to
+// browser, changes over time, doesn't necessarily fit into the
+// Patternfly color scheme, looks ugly when used together with the
+// Patternfly :focus styling, and serves no purpose when :focus is
+// already visible.
+//
+// This would all be acceptable if these elements wouldn't receive
+// :focus-visible when focused with the mouse.
+//
+// https://github.com/patternfly/patternfly/issues/4121
```

Comment:
Removing the ring is very intrusive IMO, I would rather wait for the linked issue to get an answer!

Reply:
Yes, but I have to get this out of my system. :-)